test/unit/publicapi.test.js

```diff
@@ -92,11 +92,6 @@ suite('Public API', function () {
       mq.typedText('$');
       assert.equal(mq.latex(), '\\$');
     });
-
-    test('parsing of advanced symbols', function () {
-      mq.latex('\\oplus');
-      assert.equal(mq.latex(), ''); // TODO: better LaTeX parse error behavior
```

Comment:
This seems to test that `\\oplus` doesn't work, but it does work on the Desmos branch.

Reply:
Could we change this to assert that `\\oplus` _does_ work? Or is that covered elsewhere?